.github/workflows/ci.yml

```diff
@@ -458,6 +458,9 @@ jobs:
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
           $RETRY $MAVEN clean install ${MAVEN_FAST_INSTALL} ${MAVEN_GIB} -am -pl "${{ matrix.modules }}"
+      - name: Prefetch Ryuk
+        run: |
+          $RETRY docker pull $(cat testing/trino-testing-containers/src/main/resources/io/trino/testing/containers/testcontainers-ryuk-version.txt)
```

Comment:
I think the image name is missing, only the version is being read from the file.

Reply:
thanks, fixed